packages/boot/test/bootstrapTests/test-vaults-integration.ts

```diff
@@ -226,14 +234,19 @@ test('open vault with insufficient funds gives helpful error', async t => {
   const wantMinted = giveCollateral * 100;
   const message =
     'Proposed debt {"brand":"[Alleged: IST brand]","value":"[904500000n]"} exceeds max {"brand":"[Alleged: IST brand]","value":"[63462857n]"} for {"brand":"[Alleged: ATOM brand]","value":"[9000000n]"} collateral';
+
   await t.throwsAsync(
-    wd.executeOfferMaker(Offers.vaults.OpenVault, {
-      offerId: 'open-vault',
-      collateralBrandKey,
-      giveCollateral,
-      wantMinted,
-    }),
-    { message },
+    () =>
+      wd.executeOfferMaker(Offers.vaults.OpenVault, {
+        offerId: 'open-vault',
+        collateralBrandKey,
+        giveCollateral,
+        wantMinted,
+      }),
+    {
+      message:
+        'Proposed debt {"brand":"[Alleged: IST brand]","value":"[904500000n]"} exceeds max {"brand":"[Alleged: IST brand]","value":"[63462857n]"} for {"brand":"[Alleged: ATOM brand]","value":"[9000000n]"} collateral',
+    },
```

Comment:
`const message` is defined above. If you want to inline this please remove that declaration